monitor/getBalances.js

```diff
@@ -143,6 +159,10 @@ async function main(bridgeMode) {
       }
     }
 
+    if (displayChaiToken) {
+      foreign.investedErc20Balance = Web3Utils.fromWei(investedAmountInDai)
```

Comment:
Is it possible to display a potential interests accumulated till this moment?

Reply:
Yes, used `dsrBalance() - investedAmountInDai()` for that.